src/containers/Delete/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+
+import { Button } from 'react-materialize';
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { ActionCreators } from '@actions';
+
+class Add extends Component {
+    handleButtonClick() {
+        this.props.deleteSeletedReport();
+    }
+
+    render() {
+        const selectedRate = this.props.reports.reportsList.find(
+            (val, index) => index === this.props.reports.activeIndex
+        );
+
```

Comment:
It's a good performance tweak to use find rather than filter here. Please be aware that Array.prototype.find is still considered as an "experimental" technology according to MDN docs, thus filter is commonly used in many React apps. Anyway, there's no good reason not to use find rather than filter.